src/demo/CistromeExplorer.js

```diff
@@ -110,7 +108,7 @@ const demos = {
                 {field: "Hierarchical Clustering (Average)", type: "tree", position: "right"},
                 {field: "Random 3", type: "quantitative", position: "right"},
                 {field: ["Random 1", "Random 2", "Random 3", "Random 4"], type: "quantitative", position: "right"},
-                {field: "Metadata URL", type: "url", position: "right", title: "cid"},
+                {field: "Metadata URL", type: "url", position: "right", title: "Metadata URL"},
```

Comment:
Having an option to set an alternative name to display in a track looks useful. But I think the usage of `title` is mixed between `url` type tracks and any other tracks. 

In the `url` type tracks, we were using the `title` option to set alternative names for **each sample**. For example, if `title` is set to `cid`, we were showing Cistrome IDs (e.g., `125`) instead of raw URLs (`http://dc2.cistrome.org/api/inspector?id=125`).

Since it can be misleading to use the same option for different usages depending on data types, I think we need to separate the options.

How about the following options?:
- `title`: show an alternative title of a track
- `alt`: show alternative names of individual urls

Reply:
I like the `alt` idea